media/server/main/source/ControlServerInternal.cpp

```diff
@@ -160,4 +160,9 @@ bool ControlServerInternal::registerClient(std::weak_ptr<IControlClient> client,
     setApplicationState(appState);
     return true;
 }
+void ControlServerInternal::registerLogHandler(std::shared_ptr<IClientLogHandler> &handler, bool ignoreLogLevels)
```

Comment:
Maybe this api should return a bool here and we can log an error and return false in this case, aborting the program seems quite extreme.